### DISCUSSION TITLE
Is there a way to use scaleToWidth AND scaleToHeight?

### DISCUSSION THREAD

**Discussion by tuvn1109** [fabricjs/fabric.js#7194] [Q&A] [answered]
Hello , I'm trying to draw image with given scaleX and scaleY. I use scaleToWidth and scaleToHeight (it's correct). But currently only get one scaleToWidth or scaleToHeight . How can I use these two properties at the same time even if the image is broken
```
let pugImg = new Image();
pugImg.onload = function (img) {
let pug = new fabric.Image(pugImg, {
        left: left,
        top: top,
        //      width:scaleX * width,
        //      height:scaleY * height,
});
pug.scaleToWidth(scaleX * width);
// pug.scaleToHeight(scaleY * height);
canvasDraw.add(pug);
};
pugImg.src = item;
```
My example code. I have scaleX = 0.41 and scaleY = 0.41, I want to draw the image according to that scale.
Thanks u !

**Comment by tuvn1109** [comment, marked as answer]
Hello @melchiar , nice to see you again, this question you have answered me on stackover but I have not really understood and implemented. can you help me ?

**Reply by melchiar** [reply]
@tuvn1109 I've updated my answer to your stack overflow question. Hope it helps!

https://stackoverflow.com/a/68228597/7693185

**Reply by tuvn1109** [reply]
Thank you Melchiar very much, that's what I was looking for <3

**Reply by melchiar** [reply]
No problem, feel free to mark the answer as accepted to close the issue
